<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintenanceGui</class>
 <widget class="QMainWindow" name="MaintenanceGui">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stepper Maintenance</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnOpenSerialPort">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboSelPort">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRescanPorts">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Rescan</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinMinDelay">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>500</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Step Delay (uS)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinMaxDelay">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Step Delay (uS)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWriteMinDelay">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReadMinDelay">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWriteMaxDelay">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReadMaxDelay">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStartStopMotor">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start/Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInvert">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Invert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDefault">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Default</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClear_logs">
   <property name="text">
    <string>Clear logs</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
